src/com/sap/piper/Utils.groovy

```diff
@@ -123,3 +123,20 @@ static String downloadSettingsFromUrl(script, String url, String targetFile = 's
     script.writeFile(file: targetFile, text: settings.getContent())
     return targetFile
 }
+
+/*
+ * Uses the Maven Help plugin to evaluate the given expression into the resolved values
+ * that maven sees at / generates at runtime. This way, the exact Maven coordinates and
+ * variables can be used.
+ */
+static String evaluateFromMavenPom(Script script, String pomFileName, String pomPathExpression) {
+
+    String resolvedExpression = script.mavenExecute(
+        script: script,
+        pomPath: pomFileName,
+        goals: 'help:evaluate',
+        defines: "-Dexpression=$pomPathExpression -DforceStdout -q",
```

Comment:
The parameter `-DforceStdout` [requires](https://maven.apache.org/plugins/maven-help-plugin/evaluate-mojo.html) plugin version 3.1.0+. Do we require the projects to define this version by themselves or should we specify it in this call?

Reply:
good catch, thanks

Reply:
Thanks. Did not see that!